### PR TITLE
Add Users table creation script

### PR DESCRIPTION
**Pull request by maximejimenez**
### Purpose of the PR

We want to have a Users table following the schema below : 
```
Users
- userId PRIMARY KEY
- firstName
- lastName
- fullName
- email
- birthdate
- street
- city
- country
- zip
- status ENUM (ACTIVE/LOCKED/CLOSED)
- createdAt
- updatedAt
```

I have created a script to initialize all tables at once.

### CODE REVIEW

File: src/db/users.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  firstName: SQLLITE_DATATYPE.TEXT,
+  lastName: SQLLITE_DATATYPE.TEXT,
+  fullName: SQLLITE_DATATYPE.TEXT,
+  email: SQLLITE_DATATYPE.TEXT,
+  birthdate: SQLLITE_DATATYPE.TEXT,
+  street: SQLLITE_DATATYPE.TEXT,
+  city: SQLLITE_DATATYPE.TEXT,
+  country: SQLLITE_DATATYPE.TEXT, // TODO maybe have another table for this
+  zip: SQLLITE_DATATYPE.TEXT,
+  status: SQLLITE_DATATYPE.INTEGER,
```
Comment (MatthieuLemoine) [inline src/db/users.js:21]: Why did you choose an INTEGER here?

File: src/db/users.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ * Creates the "Users" table
+ *
+ * @param {Object} db Database handle
+ */
+async function initTable(db) {
+  let tableColumns = '';
+
+  for (const key in USERS_TABLE) {
+    let column = `${key} ${USERS_TABLE[key]},`;
+    if (tableColumns !== '') {
```
Comment (MatthieuLemoine) [inline src/db/users.js:39]: There's an easier way to do that which is adding the extra space on every iteration & trim the result.

You could also use an array to store your columns & join the result

File: src/db/users.js (hunk 1 above)
Comment (maximejimenez) [reply src/db/users.js:21]: While, implementing the creation endpoint, I finally changed to `TEXT`.
I simply validate the ENUM in the swagger definition.

Comment (MatthieuLemoine) [reply src/db/users.js]: 👍

**Review by MatthieuLemoine — APPROVED** [review]

**Pull request merged**